plugins/lua/lua.c

```diff
@@ -1481,15 +1481,30 @@ static void inject_string(script_info *info, char const *line)
 {
 	lua_State *L = info->state;
 	int base, top;
+	char *program, *ret_program;
+
+	if(line[0] == '=')
+		program = g_strconcat("return ", line + 1, NULL);
+	else
+		program = g_strdup(line);
+	ret_program = g_strconcat("return ", program, NULL);
```

Comment:
Also how does this interact with multi-line lua statements. Do you need to wrap `line` in something like an IIFE and return that instead?